src/widgets.ts

```diff
@@ -18,7 +20,7 @@ type BaseOptions = {
 }
 
 type SdkPostMessage<T> = {
-  type: PostMessageTypes | "ping"
+  type: PostMessageTypes | "ping" | "mx/sdk/metrics"
```

Comment:
What do you think of `mx/sdk/...`? I'll add it to the definitions project if we like it.